src/translations/en.json

```diff
@@ -1966,10 +1967,12 @@
             "entities": {
               "name": "Entities",
               "show_header_toggle": "Show Header Toggle?",
-              "toggle": "Toggle entities."
+              "toggle": "Toggle entities.",
+              "description": "Entities will be the most common type of card that will also be the most familiar to people using the standard interface. It groups items together very close to how groups used to do."
             },
             "button": {
-              "name": "Button"
+              "name": "Button",
+              "description": "The Button card allows you to add buttons to perform tasks."
```

Comment:
This description starts with "The", while others don't, which is a bit inconsistent.